packages/slate/src/interfaces/operation.ts

```diff
@@ -228,6 +228,10 @@ export const Operation = {
           return op
         }
 
+        if (Path.isSibling(path, newPath)) {
```

Comment:
Can you please add a comment here about the reasoning? The comment below also talks about siblings, so it might be confusing to follow later.